task/09-functions-n-closures-tasks.js

```diff
@@ -65,10 +66,16 @@ function getPowerFunction(exponent) {
  *   getPolynom()      => null
  */
 function getPolynom() {
-    throw new Error('Not implemented');
+    let arg = arguments;
```

Comment:
"arguments" is deprecated for latest versions of JS. You can use 'rest parameters' instead